tools/travis.ps1

```diff
@@ -214,7 +214,6 @@ elseif($Stage -eq 'Build')
 
     if ($isFullBuild) {
         $pesterParam['Tag'] = @('CI','Feature','Scenario')
```

Comment:
Should there be a ExcludeTag for `RequireSudoOnUnix`

Reply:
The `RequireSudoOnUnix` has already been added above.
```
     $pesterParam = @{
         'binDir'     = $output
         'PassThru'   = $true
         'Terse'      = $true
         'Tag'        = @()
         'ExcludeTag' = @('RequireSudoOnUnix')
         'OutputFile' = $testResultsNoSudo
     }
```

Reply:
see line 211

Reply:
Closed